packages/cli/src/endo.js

```diff
@@ -324,80 +380,95 @@ export const main = async rawArgs => {
       }
     });
 
-  program.command('spawn <name>').action(async name => {
-    const { getBootstrap } = await provideEndoClient(
-      'cli',
-      sockPath,
-      cancelled,
-    );
-    try {
-      const bootstrap = getBootstrap();
-      await E(bootstrap).makeWorker(name);
-    } catch (error) {
-      console.error(error);
-      cancel(error);
-    }
-  });
+  program
+    .command('spawn <name>')
+    .description('creates a worker for evaluating or importing programs')
+    .action(async name => {
+      const { getBootstrap } = await provideEndoClient(
+        'cli',
+        sockPath,
+        cancelled,
+      );
+      try {
+        const bootstrap = getBootstrap();
+        await E(bootstrap).makeWorker(name);
+      } catch (error) {
+        console.error(error);
+        cancel(error);
+      }
+    });
 
-  program.command('show <name>').action(async name => {
-    const { getBootstrap } = await provideEndoClient(
-      'cli',
-      sockPath,
-      cancelled,
-    );
-    try {
-      const bootstrap = getBootstrap();
-      const pet = await E(bootstrap).provide(name);
-      console.log(pet);
-    } catch (error) {
-      console.error(error);
-      cancel(error);
-    }
-  });
+  program
+    .command('show <name>')
+    .description('prints a representation of the named value')
```

Comment:
what does "representation" add?

```suggestion
    .description('prints the named value')
```

I tried the [console.log docs](https://nodejs.org/api/console.html#consolelogdata-args); they're no help: "Prints to stdout with newline. ... the arguments are all passed to `util.format()`)"

Another angle: "carries out the named formula and prints the result".

I see `provide` in the implementation, which suggests something like: "looks the name up in a cache; carries out the named formula on cache miss; prints the result". The relationship between cache entry lifetimes and the daemon lifetime also seems relevant.